src/site/content/en/blog/how-mercadolibre-optimized-web-vitals/index.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: How Mercado Libre optimized for Web Vitals (TBT/FID)
+subhead: Optimizing interactivity of Mercado Libre's product details pages.
+authors:
+  - demianrenzulli
+  - aranhacarlos
+  - joanbaca
+date: 2020-09-01
+hero: hero.jpg
```

Comment:
The hero image looks cutoff on web.dev/blog in an awkward way:

![image](https://user-images.githubusercontent.com/4713486/92967924-f25bd900-f42e-11ea-9c9b-96a1f16bcf83.png)

Do you want to create a thumbnail that focuses on the Mercado Libre logo? https://web.dev/handbook/markup-media/#thumbnail